sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.csproj

```diff
@@ -12,11 +12,12 @@
   <ItemGroup>
     <PackageReference Include="Apache.Avro" />
     <!-- <PackageReference Include="Azure.Data.SchemaRegistry" /> -->
-    <PackageReference Include="Azure.Core" />
+    <!--<PackageReference Include="Azure.Core" />-->
```

Comment:
These project references are stuck here until we GA Azure.Data.SchemaRegistry - which shouldn't be a problem since we aren't expecting to release this library soon.